src/wallet/walletActions.js

```diff
@@ -18,9 +18,7 @@ export const getGlobalProperties = () => (dispatch) => {
   dispatch({
     type: GET_GLOBAL_PROPERTIES.ACTION,
     payload: {
-      promise: getDynamicGlobalProperties().then(result => ({
-        ...result,
-      })),
+      promise: getDynamicGlobalProperties().then(result => result),
```

Comment:
Isn't `getDynamicGlobalProperties().then(result => result)` same as `getDynamicGlobalProperties()`?

Reply:
yeah you're right, haha i think i need some coffee this morning, thanks for catching these things 👍

Reply:
just updated it now.